javascripts/discourse/initializers/initialize-composer-toggle-hijack.js

```diff
@@ -1,53 +1,240 @@
-import { withPluginApi } from "discourse/lib/plugin-api";
+import { withPluginApi } from 'discourse/lib/plugin-api'
+import discourseComputed from 'discourse-common/utils/decorators'
+
+import {
+  CREATE_SHARED_DRAFT,
+  CREATE_TOPIC,
+  EDIT,
+  PRIVATE_MESSAGE,
+  REPLY,
+} from 'discourse/models/composer'
+
+// Component can get destroyed and lose state
+let _topicSnapshot = null
+let _postSnapshot = null
+let _actionSnapshot = null
 
 export default {
-  name: "toggle-hijack",
+  name: 'toggle-hijack',
```

Comment:
Your auto formatter seems to be converting all double quotes `"` to single quotes `'`, though I believe the convention on Discourse is to have double quotes.

Also the formatting changes makes the PR more difficult to parse. I recommend reverting back to double quotes.